components/clarinet-sdk-wasm/src/core.rs

```diff
@@ -11,9 +11,9 @@ use clarinet_deployments::{
     generate_default_deployment, initiate_session_from_manifest,
     update_session_with_deployment_plan,
 };
-use clarinet_files::{
-    FileAccessor, FileLocation, ProjectManifest, StacksNetwork, WASMFileSystemAccessor,
-};
+#[cfg(target_arch = "wasm32")]
```

Comment:
`clarinet-sdk-wasm` is always and only compiled for wasm32, so that's not needed